orchestra/json_schemas/todos.py

```diff
@@ -41,7 +41,7 @@ class TodoSchema(jsl.Document):
             dictionary of attributes and predicates which get ANDed together.
     """
     id = jsl.IntField(required=True)
-    description = jsl.StringField(required=True)
+    title = jsl.StringField(required=True)
```

Comment:
This will require a data migration of the JSON objects in this field: https://github.com/b12io/orchestra/blob/main/orchestra/models/core/models.py#L623

I'll message you offline about the migration and ways to make it a little easier.